src/components/fileviewer.js

```diff
@@ -232,11 +205,11 @@ export default class FileViewerContainer extends Component {
 }
 
 
-
-class FileViewerStatus extends Component {
+/* This component contains metadata of the file */
+class FileViewerMeta extends Component {
```

Comment:
Well, if it can not be in `render()`, then we make it a class.  It should be class anyway.